.travis.yml

```diff
@@ -3,17 +3,35 @@ sudo: required
 service:
 - docker
 - docker-compose
+jobs:
+  include:
+    - stage: build and test
+      script:
+      - docker-compose -f docker-compose-dev.yml up -d --build
+      #- docker build -t todo-app-test:latest .
+      - docker run -e TRELLO_API_SECRET -e TRELLO_API_KEY -e SECRET_KEY --entrypoint poetry
+        todo-app-test:latest run pytest
+
+    - stage: docker build and publish
+      script:
+      - docker build --target production --tag $dockerUsername/todo-app-prod:latest .
+      - echo "$dockerPassword" | docker login -u "$dockerUsername" --password-stdin
+      - docker push $dockerUsername/todo-app-prod:latest
```

Comment:
If you are tagging it ":latest", then this should be part of your deploy script.

Otherwise the `todo-app-prod:latest` is extremely unstable and can refer to any branch that someone worked on recently.